syntax = "proto3";

import "buf/validate/validate.proto";

service SettingsService {
  rpc UpdateTimezone(UpdateTimezoneRequest) returns (UpdateTimezoneResponse) {}
}

message UpdateTimezoneRequest {
  string timezone = 1 [(buf.validate.field).string.max_len = 32];
}

message UpdateTimezoneResponse {}
